docs/OncoTree-Mapping-Tool.md

```diff
@@ -30,34 +29,164 @@ The list of OncoTree versions available are viewable [here](http://oncotree.mskc
+
+|PATIENT_ID|SAMPLE_ID|AGE_AT_SEQ_REPORT|ONCOTREE_CODE|ONCOTREE_CODE_OPTIONS|...|
+|-------|-------|-------|-------|-------|-------|
+|P1|S1|41||ALL -> {BLL,TLL}, more granular choices introduced|...|
+|P2|S2|60||BALL -> {BLL}, more granular choices introduced|...|
+|P3|S3|<18||TALL -> {TLL}, more granular choices introduced|...|
+|P4|S4|71|PTCL||...|
+|P5|S5|64|PTCL||...| 
+|P6|S6|36||CHL -> {CHL}, more granular choices introduced|...|
+|P7|S7|63||SpCC -> ???, Oncotee code not in source oncotree version|...|
+|P8|S8|63||MCL -> {MCL}, more granular choices introduced|...|
+|P9|S9|73||HGNEE -> ???, Oncotee code not in source oncotree version|...|
+|P10|S10|52||ONCOTREE_CODE column blank : use a valid oncotree code or "NA"|...| 
+|P11|S11|77|NA||...|   
+|P12|S12|87||TNKL -> {MTNN}, more granular choices introduced|...|
+|P13|S13|79||HIST -> {HDCN}, more granular choices introduced|...| 
+|P14|S14|53|CLLSLL||...|
+|P15|S15|69|CLLSLL||...|
+|P16|S16|65||LEUK -> {MNM}, more granular choices introduced|...|
+|P17|S17|66|MYCF||...|
+|P18|S18|66|RBL||...|                                              
+ 
+#### Step 5
+Using values in the `ONCOTREE_CODE_OPTIONS` as a guide, manually select and place an OncoTree Code in the `ONCOTREE_CODE` column. For additional information, refer to the summary file `data_clinical_sample_remapped_summary.html`. Repeat for all rows in the output file. Several examples are shown below.  
+
+ ###### Sample 1  
+
+|SAMPLE_ID|ONCOTREE_CODE  |ONCOTREE_CODE_OPTIONS|
+|:--------|:-------------:|:----------------------:|
+|S1       |	              |ALL -> {BLL,TLL}, more granular choices introduced|
+
+Source OncoTree code `ALL` maps directly to codes `BLL` and `TLL`. Users should place either `BLL` or `TLL` in the `ONCOTRE_CODE`column. The `ONCOTREE_CODE_OPTIONS` column also notes that more granular choices were introduced; as such, users can use the summary file for additional guidance. 
+
+Searching by source code, the following information can be found in the summary file:
+![ALL-example](https://raw.githubusercontent.com/averyniceday/oncotree/doc-expansion/docs/images/ALL-example.png)  
+The summary file provides a link to the closest shared parent node `LNM`; users can choose more granular nodes by referencing the provided tree:
+![LNM-tree](https://raw.githubusercontent.com/averyniceday/oncotree/doc-expansion/docs/images/LNM-tree.png)  
+
+###### Sample 2  
+
+|SAMPLE_ID|ONCOTREE_CODE  |ONCOTREE_CODE_OPTIONS|
+|:--------|:-------------:|:----------------------:|
+|S1       |	              |BALL -> {BLL}, more granular choices introduced|
+  
+Source OncoTree code `BALL` maps directly to `BLL`. Users should place `BLL` in the `ONCOTREE_CODE` column. However, similar to sample 1, the `ONCOTREE_CODE_OPTIONS` indicates there are more granular choices available. Users can follow the same steps as above and use the summary file to select a more granular node.
+ 
```

Comment:
I would add entries for all lines .. even if to say "here the oncotree code has been already filled in" or "this is like line 2"